pkg/processor/runtime/python/py/requirements/common.txt

```diff
@@ -1,3 +1,4 @@
 mock==2.0.0
 nuclio-sdk==0.5.15
 pip==24.3.1
+msgpack==1.1.0
```

Comment:
Are you sure it doesn't break anything for the older versions?

Reply:
@TomerShor nuclio sdk is the same everywhere, so should be fine as long as tests are passed